internal/testharness/pty/blackbox/stub.go

```
@@ -0,0 +1,556 @@
+	}
+	select {
+	case s.events <- struct{}{}:
+	default:
+	}
+}
+
+func boundedBody(request *http.Request) ([]byte, error) {
+	if request.ContentLength > maxHTTPBodyBytes {
+		return nil, errors.New("request body exceeds limit")
+	}
+	defer request.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `request.Body.Close` is not checked

(errcheck)